testdata/v1_1_samples/simple_router.p4

```diff
@@ -34,8 +34,8 @@ header_type ipv4_t {
         bit<8> ttl;
         bit<8> protocol;
         bit<16> hdrChecksum;
-        bit<32> srcAddr;
```

Comment:
Why is this test changed? This isn't tc case, so adding tc_type annotation doesn't look correct here